src/hashchange.js

```diff
@@ -42,14 +42,18 @@ canReflect.assign(HashchangeObservable.prototype,{
         var old = this.value;
         this.value = getHash();
         if(old !== this.value) {
-            queues.enqueueByQueue(this.handlers.getNode([]), this, [this.value, old]
-                //!steal-remove-start
-                /* jshint laxcomma: true */
-                , null
-                , [ canReflect.getName(this), "changed to", this.value, "from", old ]
-                /* jshint laxcomma: false */
-                //!steal-remove-end
-            );
+        	if(process.env.NODE_ENV !== 'production') {
+	            queues.enqueueByQueue(this.handlers.getNode([]), this, [this.value, old]
```

Comment:
This still needs to be called in production. It's just the arguments in the remove-start/stop that we don't want in prod.